src/components/Schema/Schema.tsx

```diff
@@ -6,32 +6,34 @@ import PropTypes from 'prop-types';
 import * as React from 'react';
 import { FaLink as LinkIcon } from 'react-icons/fa';
 
+import { inferType } from '../../utils/json';
 import { formatSchemaName } from '../../utils/openapi';
-import Example from './Example';
-import { IProperty } from './Property';
-import { Service } from './ServiceTags';
+import Example from '../Example';
+import SchemaContext from './SchemaContext';
 import Tag from './Tag';
 
 import styles from './Schema.module.scss';
 
 interface ISchema {
   children: React.ReactElement | React.ReactElement[];
-  example?: string,
+  json: Json;
+  jsonPointer: string;
   name: string;
-  services?: '*' | Service[];
-  type: 'array<object>' | 'object';
+  services?: MinFraudServices;
+  type?: SchemaType;
 }
 
 const slugger = new GithubSlugger();
 
-const isPropertyComponent = (child: any) => (
-  // First option for Jest renderer. Second for MDX Runtime.
-  child.type.name === 'Property' || child.props.mdxType === 'Property'
-);
-
 const Schema: React.FC<ISchema> = (props) => {
-  const { children, name, services, type } = props;
-  let { example } = props;
+  const {
+    children,
+    json,
+    jsonPointer,
+    name,
+    services,
+    type,
+  } = props;
 
   const formattedSchemaName = React.useMemo(
```

Comment:
Here too, I don't think we're getting any benefit from `useMemo`

Reply:
I removed `useMemo` where I could.